File: mass-store-open/mass_store_open_test.py
import unittest

import mass_store_open


class TestSolve(unittest.TestCase):
    def test1(self):
        p = mass_store_open.Problem(1000, 1000, 50, 3, [25, 500, 80])
        self.assertEqual(1, mass_store_open.solve(p))

    def test2(self):
        p = mass_store_open.Problem(2556, 3424, 77, 3, [137, 721, 984, 999])
        self.assertEqual(1, mass_store_open.solve(p))


if __name__ == "__main__":
    unittest.main()

File: mass-store-open/massstoreopen_test.go
package main

import (
	"testing"

	"github.com/stretchr/testify/assert"
)

func Test1(t *testing.T) {
	p := Problem{
		1000,
		1000,
		50,
		3,
		[]int{25, 500, 80},
	}
	want := 1
	got := Solve(p)
	assert.Equal(t, want, got)
}

func Test2(t *testing.T) {
	p := Problem{
		2556,
		3424,
		77,
		3,
		[]int{137, 721, 984, 999},
	}
	want := 1
	got := Solve(p)
	assert.Equal(t, want, got)
}
